pkg/scheduler/schedule_one.go

```diff
@@ -320,6 +324,7 @@ func (sched *Scheduler) skipPodSchedule(fwk framework.Framework, pod *v1.Pod) bo
 	// during its previous scheduling cycle but before getting assumed.
 	isAssumed, err := sched.Cache.IsAssumedPod(pod)
 	if err != nil {
+		// TODO: pass ctx into a revised HandleError
```

Comment:
Is there an issue for this? If so, please reference `TODO(#1234)`
 Also, does it need to be a context or can it be a logger?

Reply:
> Is there an issue for this? If so, please reference TODO(#1234)

Yes. I have added the issue number.

> Also, does it need to be a context or can it be a logger?

As far as I know, the function `utilruntime.HandleError` is used in many components. 
Do we have plans to decide whether to pass context or logger in the future? @pohly

Reply:
Can you open an issue to add a logger of Ctx to `HandleError`?